app/assets/javascripts/set-ga-client-id-on-form.js

```diff
@@ -0,0 +1,20 @@
+(function () {
+  'use strict'
+
+  window.GOVUK = window.GOVUK || {}
+  var GOVUK = window.GOVUK
+
+   function SetGaClientIdOnForm (options) {
+    if (!options.$form || !window.ga) { return }
```

Comment:
Is it necessary to check for `options.$form` here? Can this code be called by anything other than line 9 of `application.js`?

Reply:
It could be called by something else later, so doing an argument check makes the function a bit more robust.